gebrueder-weiss-woocommerce.php

```diff
@@ -29,6 +29,15 @@
     include_once dirname(__FILE__) . '/includes/GbWeiss.php';
 }
 
+/**
+ * Use Dotenv to set required environment variables and load .env file in root
+ */
+$dotenv = \Dotenv\Dotenv::createUnsafeImmutable(__DIR__);
+if (file_exists(__DIR__ . '/.env')) {
+    $dotenv->load();
+    $dotenv->required(['GEBRUEDER_WEISS_OAUTH_URL']);
+}
```

Comment:
Why do you use the `createUnsafeImmutable` method instead of `createImmutable`?
Instead of using `load` method, you can use `safeLoad`. This method does not throw an exception if the `.env` file does not exist.